src/Bart/Jenkins/Job.php

```diff
@@ -95,13 +110,13 @@ public function start(array $build_params = array())
 			));
 
 		// This gives back the general information about job
-		$metadata = $this->get_json(array());
+		$metadata = $this->getJson(array());
 
 		if ($metadata['inQueue'] > 0)
 		{
 			// Build is queued, but must wait and hasn't been assigned a number
-			$this->my_build_id = $metadata['nextBuildNumber'];
-			$this->logger->debug('Queued build: ' . $this->my_build_id);
+			$this->myBuildData = $metadata['nextBuildNumber'];
```

Comment:
Is it a number or is it data? Data to me implies something more generic than a number.